packages/common/pipes/file/file-type.validator.ts

```diff
@@ -0,0 +1,17 @@
+import { FileValidator } from './file-validator.interface';
+
+export type FileTypeValidatorOptions = {
```

Comment:
```suggestion
export type FileTypeValidatorOptions = {
```
out of curiosity, why type not interface?

Reply:
No strong reason to be honest, usually I tend to use types instead of interfaces when I don't have the intention of using the  `implements` somewhere else, and simply want to reference it. This is such case I suppose, but if you think I should change it to interface for consistency I can do that!